### PR TITLE
update MariaDB to 10.3 stable

### PR DESCRIPTION
**Pull request by it-can**

### CODE REVIEW

File: scripts/install-maria.sh [hunk 1]
```diff
@@ -31,20 +31,20 @@ rm -rf /etc/mysql
 # Add Maria PPA
 
 sudo apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 0xF1656F24C74CD1D8
-sudo add-apt-repository 'deb [arch=amd64,i386,ppc64el] http://ftp.osuosl.org/pub/mariadb/repo/10.2/ubuntu xenial main'
+sudo add-apt-repository 'deb [arch=amd64,i386,ppc64el] http://ftp.osuosl.org/pub/mariadb/repo/10.3/ubuntu xenial main'
```
Comment (svpernova09) [inline scripts/install-maria.sh]: We should be using `bionic` instead of `xenial`.

Comment (it-can) [reply scripts/install-maria.sh]: Ok wil change asap

**Comment by it-can** [comment]
@svpernova09 ok changed it

**Comment by svpernova09** [comment]
Thanks!

**Pull request merged**